rust-toolchain.toml

```diff
@@ -1,2 +1,3 @@
 [toolchain]
-channel = "1.82.0"
```

Comment:
Lock the version to 1.82.0. There are many lint errors if we upgrade to the latest stable Rust version.